src/pages/node/configuration.tsx

```diff
@@ -166,6 +187,83 @@ function SettingsPage() {
+                      checked={localNotificationSettings?.channels}
+                      onChange={() => {
+                        if (localNotificationSettings) {
+                          set_localNotificationSettings({
+                            ...localNotificationSettings, channels: !localNotificationSettings.channels,
+                          })
+                        }
+                      }}
+                      color="primary"
+                    />{' '}True
+                  </div>  
+                  <div>
+                  Message: False
+                    <Switch
+                      checked={localNotificationSettings?.message}
+                      onChange={() => {
+                        if (localNotificationSettings) {
+                          set_localNotificationSettings({
+                            ...localNotificationSettings, message: !localNotificationSettings.message,
+                          })
+                        }
+                      }}
+                      color="primary"
+                    />{' '}True
+                  </div>  
+                  <div>
+                  Node Balance: False
+                    <Switch
+                      checked={localNotificationSettings?.nodeBalances}
+                      onChange={() => {
+                        if (localNotificationSettings) {
+                          set_localNotificationSettings({
+                            ...localNotificationSettings, nodeBalances: !localNotificationSettings.nodeBalances,
+                          })
+                        }
+                      }}
+                      color="primary"
+                    />{' '}True
+                  </div>  
+                  <div>
+                  Node Info: False
+                    <Switch
+                      checked={localNotificationSettings?.nodeInfo}
+                      onChange={() => {
+                        if (localNotificationSettings) {
+                          set_localNotificationSettings({
+                            ...localNotificationSettings, nodeInfo: !localNotificationSettings.nodeInfo,
+                          })
+                        }
+                      }}
+                      color="primary"
+                    />{' '}True
+                  </div>  
+                  <div>
+                  Safe Transactions: False
+                    <Switch
+                      checked={localNotificationSettings?.pendingSafeTransaction}
+                      onChange={() => {
+                        if (localNotificationSettings) {
+                          set_localNotificationSettings({
+                            ...localNotificationSettings, pendingSafeTransaction: !localNotificationSettings.pendingSafeTransaction,
+                          })
+                        }
+                      }}
+                      color="primary"
+                    />{' '}True
+                  </div>  
```

Comment:
Do we get those in the Node Admin?

Reply:
probably not because they need to have a selectedSafe.